Project/core/Payment.cs

```diff
@@ -6,64 +6,41 @@
 
 namespace Project
 {
+    [Serializable]
     class Payment
```

Comment:
To miała być klasa abstrakcyjna
```suggestion
    abstract class Payment
```